rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -258,19 +300,30 @@ def get_performance(
             to_query_indices = all_validator_indices
 
         if now - to_ts <= DAY_IN_SECONDS:
+            _, accumulating_validators = dbeth2.group_validators_by_type(
+                database=self.database,
+                validator_indices=to_query_indices,
+            )
             balances = self.beacon_inquirer.get_balances(
                 indices_or_pubkeys=list(to_query_indices),
                 has_premium=has_premium_check(self.premium),
             )
             for pubkey, balance in balances.items():
-                entry = pnls[pubkey_to_index[pubkey]]
+                entry = pnls[v_index := pubkey_to_index[pubkey]]
                 if 'exits' in entry:
                     continue  # no outstanding balance for exits
 
                 if balance.amount == ZERO:
                     continue
 
-                outstanding_pnl = balance.amount - FVal(32)
+                if v_index in accumulating_validators:
+                    if balance.amount < 2048:
+                        continue  # no outstanding balance for accumulating validators with balance less than 2048  # noqa: E501
+                    effective_balance = FVal('2048')
+                else:
+                    effective_balance = INITIAL_ETH_DEPOSIT
+
+                outstanding_pnl = balance.amount - effective_balance
```

Comment:
Do not use wrong terms. As this can get confusing. Rename `effective_balance` here and above to max balance

```suggestion
                outstanding_pnl = balance.amount - max_balance
```